GrowthBook/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+using System.Runtime.CompilerServices;
```

Comment:
I suggest removing this file altogether and stick with the portable configuration in csproj:

```csharp
   <ItemGroup>
    <InternalsVisibleTo Include="GrowthBook.Tests" />
  </ItemGroup>
```

Reply:
Good idea, forgot about that. Done.